.github/workflows/main.yml

```diff
@@ -0,0 +1,11 @@
+name: A workflow for my Hello World file
+on: push
+jobs:
+  build:
+    name: Hello world action
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Using a standard action, you get code checked out.  This is typically the first step.